libraries/3rdparty/mbedtls/include/mbedtls/config.h

```diff
@@ -3026,8 +3029,18 @@
 
 /* To Use Function Macros MBEDTLS_PLATFORM_C must be enabled */
 /* MBEDTLS_PLATFORM_XXX_MACRO and MBEDTLS_PLATFORM_XXX_ALT cannot both be defined */
-//#define MBEDTLS_PLATFORM_CALLOC_MACRO        calloc /**< Default allocator macro to use, can be undefined */
-//#define MBEDTLS_PLATFORM_FREE_MACRO            free /**< Default free macro to use, can be undefined */
+/* Set the memory allocation functions on FreeRTOS. */
+
+#ifndef MBEDTLS_PLATFORM_CALLOC_MACRO
```

Comment:
I think it would be safer to specify these defines on the compiler/pre-processor command-line for that projects that will use it.